src/tinymce.js

```diff
@@ -80,6 +80,16 @@ angular.module('ui.tinymce', [])
               updateView(ed);
             });
 
+          // Update model on Set content
+          // Only solution I found to fix #210
+          // https://github.com/angular-ui/ui-tinymce/issues/210
+          ed.on('SetContent', function(evt) {
+            if (evt.content) {
```

Comment:
Could SetContent ever get called with an empty string and still want to update? I believe so. Instead should we just update [this block|https://github.com/angular-ui/ui-tinymce/blob/master/src/tinymce.js#L77-L81] to include SetContent?